pkg/controller/lifecycle/monitoring.go

```diff
@@ -55,45 +56,65 @@ func deployMonitoringConfig(ctx context.Context, c client.Client, namespace stri
 			return err
 		}
 
+		alertingRules := []monitoringv1.Rule{
+			{
+				Alert: "RsyslogTooManyRelpActionFailures",
+				Expr:  intstr.FromString(`sum(rate(rsyslog_pstat_failed{origin="core.action",name="rsyslg-relp"}[5m])) / sum(rate(rsyslog_pstat_processed{origin="core.action",name="rsyslog-relp"}[5m])) > bool 0.02 == 1`),
+				For:   ptr.To(monitoringv1.Duration("15m")),
+				Labels: map[string]string{
+					"service":    "rsyslog-relp",
+					"severity":   "warning",
+					"type":       "shoot",
+					"visibility": "operator",
+				},
+				Annotations: map[string]string{
+					"description": "The rsyslog relp cumulative failure rate in processing action events is greater than 2%.",
+					"summary":     "Rsyslog relp has too many failed attempts to process action events",
+				},
+			},
+			{
+				Alert: "RsyslogRelpActionProcessingRateIsZero",
+				Expr:  intstr.FromString(`rate(rsyslog_pstat_processed{origin="core.action",name="rsyslog-relp"}[5m]) == 0`),
+				For:   ptr.To(monitoringv1.Duration("15m")),
+				Labels: map[string]string{
+					"service":    "rsyslog-relp",
+					"severity":   "warning",
+					"type":       "seed",
+					"visibility": "operator",
+				},
+				Annotations: map[string]string{
+					"description": "The rsyslog relp action processing rate is 0 meaning that there is most likely something wrong with the rsyslog service.",
+					"summary":     "Rsyslog relp action processing rate is 0",
+				},
+			},
+		}
+
+		if auditConfig != nil && ptr.Deref(auditConfig.Enabled, false) {
+			alertingRules = append(alertingRules, monitoringv1.Rule{
+				Alert: "RsyslogRelpAuditRulesNotLoadedSuccessfully",
+				Expr:  intstr.FromString(`absent(rsyslog_augenrules_load_success == 1)`),
+				For:   ptr.To(monitoringv1.Duration("15m")),
+				Labels: map[string]string{
+					"service":    "rsyslog-relp",
+					"severity":   "warning",
+					"type":       "seed",
+					"visibility": "operator",
```

Comment:
I am not sure about the visibility. This should be rather `owner` or `all` as now Shoot owners can specify auditd rules via ConfigMap that gets referenced from the Shoot spec.

Reply:
I think `all` makes sense as we might also be interested in seeing these alerts if shoot owners miss them.